configtx/example_test.go

```diff
@@ -352,6 +352,33 @@ func Example_organizations() {
 	o.RemoveOrganization("OrdererOrg2")
 }
 
+// This example shows updating the individual orderer configuration values.
+func ExampleOrdererGroup_SetConfiguration_individual() {
+	baseConfig := fetchChannelConfig()
+	c := configtx.New(baseConfig)
+	o := c.Orderer()
+
+	err := o.BatchSize().SetMaxMessageCount(500)
+	if err != nil {
+		panic(err)
+	}
+
+	err = o.AddConsenter(orderer.Consenter{Address: orderer.EtcdAddress{
```

Comment:
```suggestion
	err = o.AddConsenter(orderer.Consenter{
	    Address: orderer.EtcdAddress{
```
Formatting nit: new line for Address